### PR TITLE
Update CWE to match NIST

### PR DESCRIPTION
**Pull request by lordoxley**

### CODE REVIEW

File: 2021/23xxx/CVE-2021-23872.json [hunk 1]
```diff
@@ -68,6 +68,12 @@
                         "value": "CWE-269: Improper Privilege Management"
```
Comment (cve-team) [inline 2021/23xxx/CVE-2021-23872.json:68]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/23xxx/CVE-2021-23872.json:68]: ("Expecting ',' delimiter: line 71 column 17 (char 2397)",)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/23xxx/CVE-2021-23872.json (hunk 1 above)
Comment (lordoxley) [reply 2021/23xxx/CVE-2021-23872.json:68]: Updated in second pull on same branch

Comment (lordoxley) [reply 2021/23xxx/CVE-2021-23872.json:68]: Updated formatting in second commmit

**Pull request merged**